scripts/gh_scripts/issue_comment_bot.py

```diff
@@ -217,7 +217,9 @@ def comment_on_thread(message: str):
             if lead['leadLabel'] == i['lead_label']
         )
         slack_id = next(
```

Comment:
This can still throw a `StopIteration` ; `next` throws if the generator it is supplied is empty. If you want this to return `None` in that case, you should pass in the second argument which lets you set a default value and prevents it from throwing :+1: